engine/core/src/main/java/org/datacleaner/descriptors/ClasspathScanDescriptorProvider.java

```diff
@@ -611,7 +611,7 @@ public ClasspathScanDescriptorProvider addAnalyzerClass(Class<? extends Analyzer
             try {
                 descriptor = Descriptors.ofAnalyzer(clazz);
                 _analyzerBeanDescriptors.put(clazz.getName(), descriptor);
-            } catch (Exception e) {
+            } catch (Throwable e) {
```

Comment:
Well, I encountered a case where analyzing a transformer/analyzer class with reflection caused NoClassDefFoundError. It was actually because some .jar files on classpath were in older version because of dependencies conflicts.
Your notice  is generally good, but in this case no recovery is done. DataCleaner just logs a problem that it cannot create descriptor for a component and goes on. Logging also errors is ok in this case in my opinion.